docs/guide/http_cache.md

```diff
@@ -775,36 +795,35 @@ See [Tagging from code](https://foshttpcachebundle.readthedocs.io/en/2.8.0/featu
 4\. Use deprecated `X-Location-Id` header.
 
 For custom or eZ controllers (e.g. REST) still using `X-Location-Id`, `XLocationIdResponseSubscriber` handles translating
-this header to tags for you. It supports singular and comma separated Location ID value(s):
+this header to tags. It supports singular and comma-separated Location ID value(s):
 
 ```php
 /** @var \Symfony\Component\HttpFoundation\Response $response */
 $response->headers->set('X-Location-Id', 123);
 
-// Alternatively using several location id values
+// Alternatively using several Location ID values
 $response->headers->set('X-Location-Id', '123,212,42');
 ```
 
 !!! caution "X-Location-Id use is deprecated"
 
-    `X-Location-Id` is deprecated and will be removed in future version. For rendering content it is advised to refactor
-    to use Content View, if not applicable `ContentTagInterface` or lastly manually output tags.
+    `X-Location-Id` is deprecated and will be removed in future.
+    For rendering content it is advised to refactor to use Content View,
```

Comment:
```suggestion
    For rendering content, it is advised to refactor to use Content View,
```

Also, the whole sentence seems like missing verbs to connect the nouns, esp. the second part.